admin/lib/generators/solidus_admin/install/install_generator.rb

```diff
@@ -24,6 +24,19 @@ def ignore_tailwind_build_files
       def build_tailwind
         rake "solidus_admin:tailwindcss:build"
       end
+
+      def install_lookbook
+        # The dummy app has no Gemfile
+        return unless in_root { File.exist?("Gemfile") }
+
+        gem_group :development, :test do
+          gem "lookbook"
+          gem "listen"
+          gem "actioncable"
+        end
+
+        route "mount Lookbook::Engine, at: '/lookbook' if Rails.env.development?"
```

Comment:
That will affect every `solidus_admin` installation unless I'm missing something. What about controlling through an environment variable and we set it in the sandbox script?

Reply:
That's a good idea!

Reply:
I think it's fine to install it anyway, and the starter should use it as well, that said I added a class option with the default dependant on the `SOLIDUS_ADMIN_LOOKBOOK` env variable, and changed the sandbox script accordingly.